code/controllers/master.dm

```diff
@@ -60,14 +60,17 @@ GLOBAL_REAL(Master, /datum/controller/master) = new
 
 /datum/controller/master/New()
 	// Highlander-style: there can only be one! Kill off the old and replace it with the new.
-	subsystems = list()
-	if (Master != src)
```

Comment:
Removed this line because it should be literally impossible to fail this check at this point, right @MrStonedOne?